TeamSnapSDK/SDK/DataTypes/TSDKTeamFee.m

```diff
@@ -16,4 +16,12 @@ + (NSString *)SDKType {
     return @"team_fee";
 }
 
+- (NSString *)teamFeeDescription {
```

Comment:
I tried to cherry-pick but I am getting a bad-object error. Can we hold off and remember to fix these methods? There is nothing broken with them at the moment.